format-specs/parquet-raster.md

```diff
@@ -0,0 +1,133 @@
+| `length`          | int64                                     | Length of the `data` byte_array in bytes.                                                                                                                                                                                                                                                                      |
+| `data`            | byte_array                                | Raster band pixel data (see below).                                                                                                                                                                                                                                                                            |
+
+### In-DB pixel data encoding
+
+This encoding is used when `isOffline` flag is false.
+
+| Name         | Type            | Meaning |
+|--------------|-----------------|---------|
+| `pix[w*h]`   | 1 to 8 bytes (depending on `pixtype` [1]) | Pixel values, row after row. `pix[0]` is the upper-left, `pix[w-1]` is the upper-right. <br><br>Endianness is specified at the start of WKB. It is implicit up to 8 bits (bit-order is most significant first). |
+
+### Out-DB pixel data encoding
+
+This encoding is used when `isOffline` flag is true.
+
+| Name         | Type   | Meaning                                                                 |
+|--------------|--------|-------------------------------------------------------------------------|
+| `bandNumber` | int8   | 0-based band number to use from the set available in the external file. |
+| `length`     | int16  | Length of the `url` string in bytes.                                    |
+| `url`        | string | The URI of the out-db raster file (e.g., GeoTIFF files).                |
+
+The allowed URI schemes are:
+* `file://`: Local file system
+* `http://`: HTTP
+* `https://`: HTTPS
+
+---
+
+[1] Note: 1, 2, and 4 bit `pixtype`s are still encoded as 1 byte per value.
+
+### CRS Customization
+
+CRS is represented as a string value. Writer and reader implementations are
+responsible for serializing and deserializing the CRS, respectively.
+
+As a convention to maximize the interoperability, custom CRS values can be
+specified by a string of the format `type:value`, where `type` is one of
+the following values:
+
+* `srid`: [Spatial reference identifier](https://en.wikipedia.org/wiki/Spatial_reference_system#Identifier), `value` is the SRID itself.
+* `projjson`: [PROJJSON](https://proj.org/en/stable/specifications/projjson.html), `value` is the PROJJSON string.
+
+
+## Metadata
+
+Parquet Raster files include additional metadata at two levels:
+
+1. File metadata indicating things like the version of this specification used
+2. Column metadata with additional metadata for each raster column
```

Comment:
We need to at least specify the key for storing these metadata. For instance, "A GeoParquet file MUST include a `geo.raster` key in the Parquet metadata".